spec/support/features/idv_helper.rb

```diff
@@ -20,7 +20,7 @@ def fill_out_idv_form_ok
     fill_in 'profile_zipcode', with: '66044'
     fill_in 'profile_dob', with: '01/02/1980'
     fill_in 'profile_ssn', with: '666-66-1234'
-    select "Driver's license", from: 'profile_state_id_type'
+    # click_on 'profile_state_id_type_drivers_permit'
```

Comment:
I CANNOT get this to work and I do't know why. The specs pass because there is a default, but i've tried the ids, names, label text, and can't get it to find the element.

Reply:
open to suggestions. It works here https://github.com/18F/identity-idp/pull/2201/files#diff-5f3dee3496e2e002f8694a5fbd943956R43 ...

Reply:
I dug into this one. Since we hide the radio button and use the fancy custom indicator thing, Capybara thinks the input is hidden. What you can do as a workaround is click on the label, which is what the user would actually be doing:

```ruby
find("label[for='profile_state_id_type_drivers_permit']").click
```